src/convertToObject.js

```diff
@@ -6,7 +6,36 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const newStyles = sourceString.trim();
+  const stylesArray = newStyles.split(';');
+  const arr = [];
+
+  for (let i = 0; i < stylesArray.length; i++) {
+    if (stylesArray[i].trim().length !== 0) {
+      arr.push(stylesArray[i].replace('\n', ''));
+    }
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. The for loop here can be replaced with `filter` to exclude empty styles and `map` to trim each style.